quinn/src/tests.rs

```diff
@@ -209,6 +209,131 @@ fn read_after_close() {
     runtime.run().unwrap();
 }
 
+#[test]
+fn zero_rtt() {
+    let mut endpoint = Endpoint::builder();
+
+    let log = logger();
```

Comment:
Maybe we should extract some of this stuff into a helper method? So much boilerplate...